src/controllers/customercontroller.js

```diff
@@ -22,6 +21,13 @@ class CustomerController extends OrderController
             } else {
                 toastr.warning(Controller.l("You don't have all the products to complete this order."))
             }
+        })
+
+        $("button[data-customer="+ id +"].customer-send-away").click(function (e) {
+            let customer = GAME.model.customers.filter((customer) => customer.id == id).shift();
```

Comment:
Perhaps put this function body in a closure, and re-use it both for `serve` and `send-away`.